src/components/global-header/global-header.pw.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import {
+  FullMenuExample,
+  GlobalHeaderWithErrorHandler,
+  GlobalHeaderWithLogo,
+} from "./component.test-pw";
+import navigationBar from "../../../playwright/components/navigation-bar";
+import {
+  globalHeader,
+  globalHeaderLogo,
+} from "../../../playwright/components/global-header";
+import { checkAccessibility } from "../../../playwright/support/helper";
+
+test.describe("Global Header component", () => {
+  test("should not cause a ResizeObserver-related error to occur", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<GlobalHeaderWithErrorHandler />);
+
+    await page.waitForTimeout(500);
+
+    const errorState = page.locator("#error-div");
+    await expect(errorState).toHaveText("");
+  });
+
+  test("should check that z-index of component is greater than that of NavigationBar", async ({
```

Comment:
`test("should render with z-index of component greater than z-index of NavigationBar"`